nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-test-utils/pom.xml

```diff
@@ -65,9 +65,9 @@ language governing permissions and limitations under the License. -->
             <!-- testcontainers requires elements JUnit 4 (even when running in JUnit 5 mode), expected to be removed in 2.x
                 see https://github.com/testcontainers/testcontainers-java/issues/970
              -->
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <scope>compile</scope>
+            <groupId>org.junit.vintage</groupId>
+            <artifactId>junit-vintage-engine</artifactId>
+            <scope>test</scope>
```

Comment:
IIRC, that was set as compile because it doesn't come in as a transitive dependency when `nifi-elasticsearch-test-utils` is imported by another project.

Reply:
If that happens, the tests may silently not run because there's no engine.

Reply:
Good point, although if that is case, then we should add the dependency to referencing modules, since this module does not appear to have any direction dependencies on JUnit 4. I will look at making some adjustments.